__tests__/app.test.js

```diff
@@ -0,0 +1,71 @@
+const request = require("supertest");
+const app = require("../app/app");
+const db  = require("../db/connection");
+const dataTest = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(dataTest);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('GET commands', () => {
+     test('200 status: GET /api/categories should return an array of categories ', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body.Categories)).toBe(true);
+            expect(response.body.Categories[0]).toHaveProperty("slug");
+            expect(response.body.Categories[0]).toHaveProperty("description")
```

Comment:
These assertions are simply checking only the first category, i would recommend iterating through the categories array and checking every category is what we expect.